contents/handbook/company/team.mdx

```diff
@@ -540,90 +540,90 @@ In terms of cycling– a friend once convinced me to go on a bike ride from San
 ## Contributors
 
 <div class="contributor-faces">
-  <a href="https://github.com/bhavish-agarwal"><img src="https://avatars.githubusercontent.com/u/14195048?v=4" title="bhavish-agarwal" width="50" height="50"></a>
```

Comment:
Changed those lines to fix the following error that I got when converting the file from `.md` to `.mdx`:
```
  SyntaxError: unknown: Expected corresponding JSX closing tag for <img> (282:158)
    280 | <h2>{`Contributors`}</h2>
    281 | <div class="contributor-faces">
  > 282 |   <a href="https://github.com/bhavish-agarwal"><img src="https://avatars.githubusercontent.com/u/14  195048?v=4" title="bhavish-agarwal" width="50" height="50"></a>
        |                                                                                                                                                               ^
    283 |   <a href="https://github.com/Tannergoods"><img src="https://avatars.githubusercontent.com/u/60791437?v=f" title  [33m="Tannergoods" width="50" height="50"></a>
    284 |   <a href="https://github.com/ungless"><img src="https://avatars.githubusercontent.com/u/8397061?v=4" title=  "ungless" width="50" height="50"></a>
    285 |   <a href="https://github.com/gzog"><img src="https://avatars.githubusercontent.com/u/1487006?v=4" title=  9m"gzog" width="50" height="50"></a>/app/contents/handbook/company/team.mdx: unknown: Expected corresponding
JSX closing   tag for <img> (282:158)
    280 | <h2>{`Contributors`}</h2>
    281 | <div class="contributor-faces">
  > 282 |   <a href="https://github.com/bhavish-agarwal"><img src="https://avatars.githubusercontent.com/u/14  195048?v=4" title="bhavish-agarwal" width="50" height="50"></a>
        |                                                                                                                                                               ^
    283 |   <a href="https://github.com/Tannergoods"><img src="https://avatars.githubusercontent.com/u/60791437?v=f" title  [33m="Tannergoods" width="50" height="50"></a>
    284 |   <a href="https://github.com/ungless"><img src="https://avatars.githubusercontent.com/u/8397061?v=4" title=  "ungless" width="50" height="50"></a>
    285 |   <a href="https://github.com/gzog"><img src="https://avatars.githubusercontent.com/u/1487006?v=4" title=  9m"gzog" width="50" height="50"></a>
```